apps/vr-tests-react-components/src/stories/Menu/Menu.stories.tsx

```diff
@@ -31,7 +31,7 @@ storiesOf('Menu Converged - basic', module)
 
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29182&commitId=09dd9a20da4ea6cfbcc35bff40052d1f361ac158&env=prod&clientType=FLUENTUI)


  <details><summary><b>Menu Converged - basic</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - basic.default - RTL.chromium.png|57| Changed|
|Menu Converged - basic.default.chromium.png|57| Changed|
|Menu Converged - basic.default.hover menuitem.chromium.png|46| Changed|
|Menu Converged - basic.default - RTL.hover menuitem.chromium.png|46| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>